tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Redis/RedisHelper.cs

```diff
@@ -34,12 +34,12 @@ internal static Scope CreateScope(Tracer tracer, IntegrationId integrationId, st
                 return null;
             }
 
-            string serviceName = tracer.CurrentTraceSettings.GetServiceName(tracer, ServiceName);
+            string serviceName = tracer.CurrentTraceSettings.Schema.Database.GetServiceName(ServiceName);
             Scope scope = null;
 
             try
             {
-                var tags = new RedisTags();
+                var tags = tracer.CurrentTraceSettings.Schema.Database.CreateRedisTags();
                 tags.InstrumentationName = integrationName;
 
                 scope = tracer.StartActiveInternal(OperationName, serviceName: serviceName, tags: tags);
```

Comment:
@zacharycmontoya and @amarziali know better than me, but aren't you supposed to update operation name as well? 
I'm not adding a suggestion because I think you should rename also `ServiceName` in `DatabaseType` and remove  the variable `OperationName` (again not 100% sure but I think it's what Zach used to do)

Reply:
Regarding the operation name, checking our service naming flexibility inventory, we want to keep the operation name as `redis.command`. It would be good if @amarziali could confirm. If it stays the same (and because Redis is a special snowflake), keeping the `OperationName` constant could be a good way of handling it for now.

Though not required, and it wouldn't change the data at all, I would like to change the way the service name is retrieved. I'll post a separate suggestion for that

Reply:
Oh nevermind, the service name retrieval looks good to me 👍🏼

Reply:
the operation is ok. redis is classified as 'cache' (yes it can be a db but it has been done this way :) ) hence `.command` is the good suffix

Reply:
@zacharycmontoya do you think this should be ready to merge?